### PR TITLE
filters: implement status filter and a new view (to display all entries)

### LINKED ISSUE #1382: [v2] Filter on status entry (unread/archive/starred)
see #1380. 
We need to filter on status entry.

### PR DESCRIPTION
**Pull request by nicosomb**
fix #1382

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -3466,16 +3466,16 @@
         },
         {
             "name": "phpunit/phpunit",
-            "version": "4.8.4",
+            "version": "4.8.5",
```
Comment (j0k3r) [inline composer.lock]: Try to avoid committing composer.lock if you didn't touch the composer.json :)

Comment (nicosomb) [inline composer.lock]: Do you want I remove composer.lock?

Comment (j0k3r) [inline composer.lock]: Yep

**Comment by j0k3r** [comment]
Could you rebase your branch against the v2 branch and use the `showEntries` instead ?

**Comment by nicosomb** [comment]
Done this morning, still in my train, but not pushed after ;-)

**Pull request merged**